src/validation_workflow/rpm/validation_rpm.py

```diff
@@ -21,36 +21,40 @@ class ValidateRpm(Validation, DownloadUtils):
 
     def __init__(self, args: ValidationArgs) -> None:
         super().__init__(args)
-        self.base_url = "https://artifacts.opensearch.org/releases/bundle/"
+        self.base_url_production = "https://artifacts.opensearch.org/releases/bundle/"
+        self.base_url_staging = "https://ci.opensearch.org/ci/dbc/distribution-build-"
         self.tmp_dir = TemporaryDirectory()
 
     def download_artifacts(self) -> bool:
-        architectures = ["x64", "arm64"]
+        isFilePathEmpty = bool(self.args.file_path)
```

Comment:
nit: Since this used only once in this class you can avoid assigning a variable to it. Applies to all other 3 instances

Reply:
I used this as a flag to iterate through all conditions, as self.args.file_path will be assigned values inside the loop. I wanted to distinguish if the user provided version or file_path and frame the url only if version is given.

Reply:
I see! Thanks!